presto-tests/src/test/java/io/prestosql/tests/AbstractTestEngineOnlyQueries.java

```diff
@@ -730,7 +730,7 @@ public void testDescribeInput()
         MaterializedResult expected = resultBuilder(session, BIGINT, VARCHAR)
                 .row(0, "unknown")
                 .row(1, "bigint")
-                .row(2, "varchar")
+                .row(2, "varchar(25)")
```

Comment:
It looks like we are missing dedicated test for this feature. Can you please add it where you could also test other parametric types including timestamp, char and decimal?

Reply:
Thanks for your insight. Fixed it